packages/language-server/src/plugins/typescript/TypeScriptPlugin.ts

```diff
@@ -334,24 +336,32 @@ export class TypeScriptPlugin
         return this.findReferencesProvider.findReferences(document, position, context);
     }
 
-    onWatchFileChanges(fileName: string, changeType: FileChangeType) {
-        const scriptKind = getScriptKindFromFileName(fileName);
+    onWatchFileChanges(onWatchFileChangesParas: OnWatchFileChangesPara[]) {
+        const doneUpdateProjectFiles = new Set<SnapshotManager>();
```

Comment:
I think this can be simplified to a simple `boolean`

Reply:
The reason I use a `Set` is because there might be possibility where files belongs to different `SnapshotManager`, for example a sub directory with seperate `tsconfig.json`

Reply:
Oh right, I forgot about that case. All good then, thanks for the explanation!